SignalServiceKit/src/Network/API/Requests/OWSRequestFactory.h

```diff
@@ -14,6 +19,57 @@ NS_ASSUME_NONNULL_BEGIN
 
 + (TSRequest *)disable2FARequest;
 
++ (TSRequest *)acknowledgeMessageDeliveryRequestWithSource:(NSString *)source timestamp:(UInt64)timestamp;
```

Comment:
I've tried to keep this change as "mechanical" as possible by replacing request classes with names like this: `TSAliceRequest` with a factory method like this: `aliceRequest`, and preserving the naming and ordering of the parameters.  In a few cases I improved parameter names.